ArmaForces.Boderator/ArmaForces.Boderator.BotService/DataClasses/SingleUser.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+
+namespace ArmaForces.Boderator.BotService.DataClasses
```

Comment:
The `DataClasses` is awful and stinks with legacy `ArmaforcesMissionBot`, don't. You are creating a `DLC` feature so almost everything you do should be in `Features/DLC` folder/namespace. This one will probably go to `Models` subfolder.